timesketch/frontend-ng/src/views/Sketch.vue

```diff
@@ -15,150 +15,257 @@ limitations under the License.
-              <v-list-item-subtitle>
-                <small>{{ sketch.created_at | timeSince }} by {{ sketch.user.username }}</small>
-              </v-list-item-subtitle>
-            </v-list-item-content>
-          </v-list-item>
-
-          <v-list-item>
-            <v-list-item-content>
-              <v-list-item-title>
-                <strong>Access: </strong>
-                <span v-if="meta.permissions.public">Public</span>
-                <span v-else>Restricted</span>
-              </v-list-item-title>
-              <v-list-item-subtitle>
-                <small v-if="meta.permissions.public">Visibly to all users on this server</small>
-                <small v-else>Only people with access can open</small>
-              </v-list-item-subtitle>
-            </v-list-item-content>
-          </v-list-item>
-
-          <v-list-item>
-            <v-list-item-content>
-              <v-list-item-title>
-                <strong>Shared with</strong>
-              </v-list-item-title>
-              <v-list-item-subtitle>
-                <small>People and groups with access</small>
-              </v-list-item-subtitle>
-            </v-list-item-content>
-          </v-list-item>
-        </v-list>
-      </v-expand-transition>
-      <v-divider></v-divider>
-
-      <!-- Reusable dialog for adding a scenario -->
-      <ts-add-scenario-dialog :dialog.sync="dialog" @close-dialog="dialog = false" />
-
-      <div v-if="!scenarios && scenarioTemplates.length">
-        <div class="pa-4" flat :class="$vuetify.theme.dark ? 'dark-hover' : 'light-hover'">
-          <span @click="addScenarioDialog" style="cursor: pointer"
-            ><v-icon left>mdi-plus</v-icon> Add Investigation Scenario</span
-          >
-        </div>
-        <v-divider></v-divider>
+    <!-- Top horizontal toolbar -->
+    <v-toolbar flat color="transparent">
+      <v-avatar v-show="!showLeftPanel" class="mt-2 ml-n1">
+        <router-link to="/">
+          <v-img src="/dist/timesketch-color.png" max-height="25" max-width="25" contain></v-img>
+        </router-link>
+      </v-avatar>
+
+      <v-btn icon v-show="!showLeftPanel" @click="toggleLeftPanel" class="ml-n1">
+        <v-icon>mdi-menu</v-icon>
+      </v-btn>
+
+      <div v-if="activeContext.question" class="ml-2">
+        <strong>{{ activeContext.question.display_name }}</strong>
       </div>
 
-      <ts-scenario v-for="scenario in scenarios" :key="scenario.id" :scenario="scenario">
-        <v-menu offset-y :close-on-content-click="true">
-          <template v-slot:activator="{ on, attrs }">
+      <v-spacer></v-spacer>
+      <v-btn small depressed v-on:click="switchUI"> Use the old UI </v-btn>
+      <v-btn small depressed color="primary" class="ml-2">
+        <v-icon small left>mdi-account-multiple-plus</v-icon>
+        Share
+      </v-btn>
+      <v-avatar color="grey lighten-1" size="25" class="ml-3">
+        <span class="white--text">{{ currentUser | capitalize }}</span>
```

Comment:
Same as in the Home.vue component: The username overflows the v-avatar element. I recommend to either go with initials or dynamically adjust the v-avatar element.